docs/data-sources/tag.md

```diff
@@ -1,39 +1,77 @@
 ---
+# Documentation generated by "gen/generator.go"; DO NOT EDIT.
+# In order to regenerate this file execute `go generate` from the repository root.
+# More details can be found in the [README](https://github.com/CiscoDevNet/terraform-provider-aci/blob/master/README.md).
+subcategory: "Generic"
 layout: "aci"
 page_title: "ACI: aci_tag"
 sidebar_current: "docs-aci-data-source-aci_tag"
-subcategory: "Generic"
 description: |-
-  Data source for ACI Tag
+  Data source for Tag
 ---
 
 # aci_tag #
 
-Data source for ACI Tag
+Data source for Tag
 
 ## API Information ##
 
-* `Class` - tagTag
-* `Distinguished Name` - {parent_dn}/tagKey-{key}
+* Class: [tagTag](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/tagTag/overview)
+
+* Supported in ACI versions: 3.2(1l) and later.
+
+* Distinguished Name Formats:
+  - Too many DN formats to display, see model documentation for all possible parents of [tagTag](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/tagTag/overview).
+  - `uni/tn-{name}/BD-{name}/tagKey-{key}`
+  - `uni/tn-{name}/ap-{name}/epg-{name}/tagKey-{key}`
+  - `uni/tn-{name}/ap-{name}/tagKey-{key}`
+  - `uni/tn-{name}/eptags/epiptag-[{ip}]-{ctxName}/tagKey-{key}`
+  - `uni/tn-{name}/tagKey-{key}`
 
 ## GUI Information ##
 
-* `Location` - Under every object as Policy Tags in the Operational tab in recent APIC versions.
+* Location: `Generic`
```

Comment:
I actually like the previous location we should probably update it for something similar.

Reply:
ok do you want me to also change for annotation to similar sentence? since here we said set to generic for now?